setup.sh

```diff
@@ -7,6 +7,7 @@ sudo -n dnf install -y pipenv
 cd /vagrant
 # Install dependencies with Pipenv
 pipenv sync --dev
+pipenv run pip install django-crispy-forms 
```

Comment:
I think it will be better to add it as part of the 
falcon/Pipfile file
in this way, the install will be with all the other dependencies in line # 9